### PR TITLE
OLED Configuratoin

### PR DESCRIPTION
**Pull request by gautam-dev-maker**
This PR adds the following functionalities

* Solves the MPU6050 integration with OLED
* Add LSA bar funciont
* Add WallE logo function
* Add function to display MPU6050 readings on OLED

### CODE REVIEW

**Review by VedantParanjape — APPROVED** [review]

**Comment by VedantParanjape** [comment]
@gautam-dev-maker can you change SRA logo to be white logo on black background. The current one is opposite of that and doesn't look that good on the OLED.

File: src/oled.c [hunk 1]
```diff
@@ -80,21 +111,105 @@ esp_err_t init_oled(u8g2_t* u8g2)
...
+	u8g2_DrawStr(u8g2, 17, 30, "W a l l - E  2 0 2 2");
+
+	// Sends the buffer to the OLED Display
+	u8g2_SendBuffer(u8g2);
+	ESP_LOGI(TAG_OLED, "Displayed Wall-E logo on OLED Screen");
+	
+    return ESP_OK;
+}
+
+esp_err_t display_lsa(line_sensor_array readings,u8g2_t *u8g2)
```
Comment (VedantParanjape) [inline src/oled.c:165]:
```suggestion
esp_err_t display_lsa(line_sensor_array readings, u8g2_t *u8g2)
```

File: src/oled.c [hunk 2]
```diff
@@ -80,21 +111,105 @@ esp_err_t init_oled(u8g2_t* u8g2)
...
 {
+	// Clear the screen
+	u8g2_ClearBuffer(u8g2);
+
+	// Fill the screen
     u8g2_DrawBox(u8g2, 0, 0, 128, 64);
-    u8g2_DrawBitmap(u8g2,27,0,9,36,sra_logo);
 
+	// Draw Wall-E logo on the screen
+    u8g2_DrawBitmap(u8g2,27,0,9,36,sra_logo);
```
Comment (VedantParanjape) [inline src/oled.c:137]:
```suggestion
    u8g2_DrawBitmap(u8g2, 27, 0, 9, 36, sra_logo);
```
Even though a nitpick, please run a linter to cleanup the code formatting in many places

**Review by VedantParanjape — CHANGES_REQUESTED** [review]

**Comment by VedantParanjape** [comment]
LGTM ! @gautam-dev-maker should I merge it ?

**Comment by gautam-dev-maker** [comment]
Shouldl I make the README changes in this PR itself?

**Comment by VedantParanjape** [comment]
> Shouldl I make the README changes in this PR itself?

Nope, can be done later. Necessary to get functional changes in.

**Pull request merged**